_includes/apple-touch-icons.md

```diff
@@ -0,0 +1,9 @@
+{% comment -%}
+TODO: Figure out how to use front matter with proper arrays instead of split
+strings.
+{%- endcomment -%}
+{% assign design_guide_version_url = include.design_guide_version_url %}
+{%- assign sizes = "57, 60, 72, 76, 114, 120, 144, 152, 167, 180, 1024" | split: ',' -%}
```

Comment:
This is already an array from my understanding reading the section about `Array` in https://shopify.github.io/liquid/basics/types/

Reply:
I can't see anything there about **declaring** arrays, only how to **access** them. It seems to be presumed that data constructs like arrays in Liquid come from front matter, `_config.yml`, [data files](https://jekyllrb.com/docs/datafiles/) or [collections](https://jekyllrb.com/docs/collections/).

Reply:
I misread the text, hence the second comment. My bad.